it/python/todo/test_sample.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from kiota_abstractions.headers_collection import HeadersCollection
+from kiota_abstractions.authentication.anonymous_authentication_provider import (
+    AnonymousAuthenticationProvider,
+)
+from kiota_http.httpx_request_adapter import HttpxRequestAdapter
+
+from client.api_client import ApiClient
+from client.models.new_todo import NewTodo
+from client.todos.todos_request_builder import TodosRequestBuilder
+
+@pytest.mark.asyncio
+async def test_basic_upload_download():
+    auth_provider = AnonymousAuthenticationProvider()
+    request_adapter = HttpxRequestAdapter(auth_provider)
+    request_adapter.base_url = 'http://127.0.0.1:1080'
+    client = ApiClient(request_adapter)
+    
+    myHeaders = HeadersCollection()
```

Comment:
This is the nasty detail, it looks like a lot of boilerplate for something that we used to write as:

```py
config = TodosRequestBuilder.TodosRequestBuilderPostRequestConfiguration(
        headers = { "My-Extra-Header", "hello" }
 )
```

Either we add more helper methods to make the conversion automatic for the user or, at the very least, we should add a constructor from `Dict[str, str | List[str]]` to `HeadersCollection`.

What do you think?

Reply:
for context to others reading this. It's being discussed here https://github.com/microsoft/kiota-abstractions-python/issues/208